src/templates/partials/lighttpd.hbs

```diff
@@ -11,36 +15,59 @@ $SERVER["socket"] == ":80" {
 {{/if}}
 }
 
+#server.modules += ("mod_setenv")
+$HTTP["scheme"] == "https" {
+    # HTTP Strict Transport Security ({{output.hstsMaxAge}} seconds)
+    setenv.add-response-header = (
+        "Strict-Transport-Security" => "max-age={{output.hstsMaxAge}}"
+    )
+}
 {{/if}}
+
+#server.modules += ("mod_openssl")
 $SERVER["socket"] == ":443" {
-    ssl.engine   = "enable"
+    ssl.engine  = "enable"
 
+  {{#if (minver "1.4.53" form.serverVersion)}}
+    ssl.privkey = "/path/to/private_key"
```

Comment:
Do the equal signs still line up after all this?

Reply:
many of them do.